scripts/sanity_check.sh

```diff
@@ -642,37 +647,65 @@ check_network() {
     fi
 
     echo "---------------------- Network ------------------"
-    curl localhost:8837/v1/config/interfaces -o /tmp/scc_interfaces &>/dev/null
+    curl localhost:8837/v1/config/interfaces -s -o /tmp/scc_interfaces
```

Comment:
Maybe safer to use a unique file name like`/tmp/scc_interfaces-$$`,  just in case multiple runs of same script at the same time

Reply:
It might be safer to have a check on HTTP code of curl command , in case of any HTTP error, which might cause error output in the temporary file

```
http_code=$(curl -s -w '%{http_code}\n' 'localhost:8837/v1/config/interfaces' -o /tmp/scc_interfaces-$$)
if [[ "$http_code" != '200' ]]; then
   echo "ERROR:  HTTP error $http_code" >&2
fi
```

Reply:
this API should not return error, otherwise we should fix firerouter.
parallel runs are possible but rarely there's a config change between runs.